cmd/utils/diskusage.go

```diff
@@ -32,12 +32,5 @@ func getFreeDiskSpace(path string) (uint64, error) {
 	}
 
 	// Available blocks * size per block = available space in bytes
-	var bavail = stat.Bavail
-	if stat.Bavail < 0 {
-		// FreeBSD can have a negative number of blocks available
-		// because of the grace limit.
```

Comment:
This change is wrong, due to the comment here. On freebsd, `stat.Bavail` is defined as a signed integer

Reply:
🤣 I felt it ...